content/tutorials/sweeps.md

```diff
@@ -6,219 +6,166 @@ menu:
 
-{{< cta-button colabLink="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/pytorch/Organizing_Hyperparameter_Sweeps_in_PyTorch_with_W&B.ipynb" >}}
+In this tutorial, create a hyperparameter search using the W&B PyTorch integration. Follow along with a [video tutorial](http://wandb.me/sweeps-video).
 
-Finding a machine learning model that meets your desired metric (such as model accuracy) is normally a redundant task that can take multiple iterations. To make matters worse, it might be unclear which hyperparameter combinations to use for a given training run. 
+## Sweeps: An overview
 
-Use W&B Sweeps to create an organized and efficient way to automatically search through combinations of hyperparameter values such as the learning rate, batch size, number of hidden layers, optimizer type and more to find values that optimize your model based on your desired metric.
+Running a hyperparameter sweep with Weights & Biases consists of three steps:
 
-In this tutorial you will create a hyperparameter search with W&B PyTorch integration. Follow along with a [video tutorial](http://wandb.me/sweeps-video).
+1. **Define the sweep:** Create a dictionary or a [YAML file]({{< relref "/guides/models/sweeps/define-sweep-configuration" >}}) specifying the parameters to search, the search strategy, and the optimization metric.
+2. **Initialize the sweep:** Use one line of code to initialize the sweep with the configuration dictionary: `sweep_id = wandb.sweep(sweep_config)`.
+3. **Run the sweep personal digital assistant:** Call `wandb.agent()` with the `sweep_id` to run it with a function that defines the model architecture and trains it: `wandb.agent(sweep_id, function=train)`.
 
-{{< img src="/images/tutorials/sweeps-1.png" alt="" >}}
-
-## Sweeps: An Overview
-
-Running a hyperparameter sweep with Weights & Biases is very easy. There are just 3 simple steps:
-
-1. **Define the sweep:** we do this by creating a dictionary or a [YAML file]({{< relref "/guides/models/sweeps/define-sweep-configuration" >}}) that specifies the parameters to search through, the search strategy, the optimization metric et all.
-
-2. **Initialize the sweep:** with one line of code we initialize the sweep and pass in the dictionary of sweep configurations:
-`sweep_id = wandb.sweep(sweep_config)`
-
-3. **Run the sweep agent:** also accomplished with one line of code, we call `wandb.agent()` and pass the `sweep_id` to run, along with a function that defines your model architecture and trains it:
-`wandb.agent(sweep_id, function=train)`
-
-
-## Before you get started
-
-Install W&B and import the W&B Python SDK into your notebook:
-
-1. Install with `!pip install`:
+## Before you start
 
+Install W&B and import the W&B Python SDK in your notebook:
 
+1. Install W&B:
 ```
```

Comment:
It seems like the bot should be able to figure out the lexer to use. These also need to be indented a tab stop.